epios/post_process.py

```diff
@@ -0,0 +1,1423 @@
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+import math
+from itertools import product
+import multiprocessing
+import sys
+import os
+from epios.sampler import Sampler
+from epios.sampler_age_region import SamplerAgeRegion
+from epios.sampler_age import SamplerAge
+from epios.sampler_region import SamplerRegion
+from epios.sampling_maker import SamplingMaker
+
+
+class PostProcess():
+    '''
+    This class is to automatically sample the population at several given time points.
+
+    And generate plots and comparison with the true infection level within the population.
+
+    How to use:
+    -----------
+
+    Define an instance and input the demographical and time data of the population
+    Then use __call__ method directly to generate plots and comparison
+
+    To define an instance of PostProcess, you need the following inputs:
+
+    Parameters:
+    -----------
+
+    demo_data : pandas.DataFrame
+        The geographical data of the population
+    time_data : pandas.DataFrame
+        The infection data of the population at different time points
+
+    '''
+
+    def __init__(self, demo_data: pd.DataFrame, time_data: pd.DataFrame):
+        self.demo_data = demo_data
+        self.time_data = time_data
+
+    def __call__(self, sampling_method, sample_size, time_sample, non_responder=False, comparison=True,
+                 nonresprate=None, data_store_path='./input/', **kwargs):
```

Comment:
I just used this expression from Antonio's class - SamplingMaker
Maybe I will still keep this expression, since this variable is used in lots of places...

Reply:
I have modified `nonresprate` to `non_resp_rate`